subsys/logging/Kconfig

```diff
@@ -279,6 +279,40 @@ config LOG_FRONTEND
 	  When enabled, logs are redirected to a custom frontend instead
 	  of being processed by the logger.
 
+config LOG_BACKEND_FLASH
+	bool "Enable FLASH backend"
+	depends on FLASH
+	depends on FILE_SYSTEM_LITTLEFS
+	default n
+	help
+	  When enabled backend is using FLASH to output logs.
+
+if LOG_BACKEND_FLASH
+
+config OVERWRITE_OLD_LOG_FILES
```

Comment:
it's good to prefix kconfig options with module. It would be descriptive enough to call it `LOG_BACKEND_FLASH_OVERWRITE`

Reply:
Macro renamed to `LOG_BACKEND_FS_OVERWRITE` following suggestion from another comment.